<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecentBooksDlg</class>
 <widget class="QDialog" name="RecentBooksDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Recent books</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Recent books</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblPosition">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btnFolder">
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="cr3.qrc">
         <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemove">
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="cr3.qrc">
         <normaloff>:/trash.png</normaloff>:/trash.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="NavigationBar" name="navBar" native="true">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnOk">
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="cr3.qrc">
         <normaloff>:/book.png</normaloff>:/book.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCancel">
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="cr3.qrc">
         <normaloff>:/close.png</normaloff>:/close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavigationBar</class>
   <extends>QWidget</extends>
   <header>navigationbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cr3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>RecentBooksDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
